.github/workflows/deploy-prod-updates.yml

```diff
@@ -34,3 +34,20 @@ jobs:
           aws-secret-access-key: ${{ secrets.UPDATES_PROD_AWS_SECRET_ACCESS_KEY }}
           aws-region: us-west-2
       - run: aws s3 sync --acl public-read rumba-updates/ s3://updates-prod-developer-mozilla-e2b95ed55d379b1a/rumba-bcd-updates/
+  trigger-prod-rumba-update-if-new:
+    name: Trigger rumba updates if new release.
+    needs: deploy-prod-updates
+    runs-on: ubuntu-latest
+    env:
+      RUMBA_AUTH: ${{ secrets.RUMBA_PROD_API_KEY }}
+    steps:
+      - run: |
+          most_recent_update=`curl https://updates.developer.mozilla.org/rumba-bcd-updates/bcd-updates.json | jq '[.browsers[].releases[] | select(.status == "current")] | max_by(.release_date) | .release_date'`
+          most_recent_api_release=`curl https://developer.mozilla.org/api/v2/updates/ | jq '.data[0].release_date'`
+          if [ $(date -d $most_recent_update +%s) -gt $(date -d $most_recent_api_release +%s) ]; 
+          then 
+            echo "$most_recent_update is more recent than $most_recent_api_release";
+            curl -X POST 'https://developer.mozilla.org/admin-api/v2/updates/' --header "Authorization: Bearer $RUMBA_AUTH"
```

Comment:
can we do `curl --fail` to get it to fail the job if this returns an internal server error?

Reply:
Resolved via a16267808705afd93514bc6cdd218bf594ff55d8.